### PR TITLE
Subscribe to orderbook updates in debug ui

### PR DESCRIPTION
**Pull request by brittcyr**

### CODE REVIEW

File: debug-ui/app/components/Orderbook.tsx [hunk 1]
```diff
@@ -17,33 +16,56 @@ const Orderbook = ({
...
 }): ReactElement => {
+  const [marketData, setMarketData] = useState<Buffer>();
   const [bids, setBids] = useState<RestingOrder[]>([]);
   const [asks, setAsks] = useState<RestingOrder[]>([]);
   const [currentSlot, setCurrentSlot] = useState<number>(0);
 
   const { connection: conn } = useConnection();
   const { wallet } = useWallet();
 
+  conn.onAccountChange(
```
Comment (TovarishFin) [inline debug-ui/app/components/Orderbook.tsx:27]: much nicer approach. especially since we don't care about reliability so much here.

**Review by TovarishFin — APPROVED** [review]

File: debug-ui/app/components/Orderbook.tsx (hunk 1 above)
Comment (mschneider) [reply debug-ui/app/components/Orderbook.tsx:27]: you need to wrap this inside an effect, you cant just open a new subscription on every component render

Comment (mschneider) [reply debug-ui/app/components/Orderbook.tsx:27]: also this subscription will persist on the global connection object that you pull in via useConnection if you switch markets

File: debug-ui/app/components/Orderbook.tsx [hunk 2]
```diff
@@ -25,25 +25,61 @@ const Orderbook = ({
...
+    const updateSlot = async (): Promise<void> => {
+      try {
+        setCurrentSlot(await conn.getSlot());
+      } catch (err) {
+        console.error('getSlot:', err);
+      }
+    };
+    updateSlot();
+    // 200 ms is half a slot, so we can expect this to update on each slot.
+    const id = setInterval(updateSlot, 200);
```
Comment (mschneider) [inline debug-ui/app/components/Orderbook.tsx:82]: could use a slot subscription, 5rps for get slot from a public UI can quickly become a few 100rps

**Review by mschneider — APPROVED** [review]

**Pull request merged**